modules/jaeger-config-storage.adoc

```diff
@@ -73,10 +73,10 @@ When the `storage:type` is set to `elasticsearch` but there is no value set for
 
 [NOTE]
 ====
-If you already have installed Elasticsearch as part of OpenShift logging, the Jaeger Operator can use the installed Elasticsearch Operator to provision storage.
+If you already have installed Elasticsearch as part of OpenShift logging, the Jaeger Operator can use the installed OpenShift Elasticsearch Operator to provision storage.
 ====
 
-The following configuration parameters are for a _self-provisioned_ Elasticsearch instance, that is an instance created by the Jaeger Operator using the Elasticsearch Operator.  You specify configuration options for self-provisioned Elasticsearch under `spec:storage:elasticsearch`  in your configuration file.
+The following configuration parameters are for a _self-provisioned_ Elasticsearch instance, that is an instance created by the Jaeger Operator using the OpenShift Elasticsearch Operator.  You specify configuration options for self-provisioned Elasticsearch under `spec:storage:elasticsearch`  in your configuration file.
```

Comment:
`.  ` here as well

Reply:
Updated this one plus about sixteen others in this file. Thanks.